nodeJS/express/installing_postgresql.md

```diff
@@ -172,110 +172,38 @@ If your terminal doesn't recognize `brew`, then you'll need to go and install ho
 
 ### Step 2: Install the PostgreSQL packages
 
-Now that we've ensured our packages are up to date, we will use brew to install PostgreSQL.
+Installing PostgreSQL via Postgres.app is simple. Visit [Postgres.app](https://postgresapp.com/) and follow the instructions outlined. Importantly, you'll want to configure your `$PATH` so you can access the tooling that comes along with PostgreSQL.
 
-```bash
-brew install postgresql@14
-```
-
-After installation is complete, let's start the server using this command:
+You can run this command to update your path:
 
-```bash
-brew services start postgresql@14
+  ```bash
+sudo mkdir -p /etc/paths.d &&
+echo /Applications/Postgres.app/Contents/Versions/latest/bin | sudo tee /etc/paths.d/postgresapp
 ```
 
-If you are unsure about whether `postgresql` is active, it's possible to check with this command:
+After you've run this command and restarted your terminal, you can run `which psql` and we would expect this output:
 
 ```bash
-brew services info postgresql@14
+/Applications/Postgres.app/Contents/Versions/latest/bin/psql
 ```
 
-Got an error, or don't see an active service? Come visit the [Discord](https://discord.gg/V75WSQG) for some help!
-
-If the `postgresql` service is active, move on to the next step.
-
-### Step 3: Setting up PostgreSQL
-
-PostgreSQL is now running, but we have to configure it in order to be able to use it with our local Express applications.
+If you don't see a similar output, come visit [our Discord server](https://discord.gg/fbFCkYabZB) for some help!
 
-#### 3.1 PostgreSQL roles
+Please note that that Postgres.app installs servers without any security configured, and will accept all local connections without a password. This is fine for our development environment, but be aware that when you move things to production, this will not be a good strategy.
```

Comment:
Nit: Can we wrap this in a warning box - just to make sure it's harder to miss.

Reply:
Thanks, @KevinMulhern ! I have additional changes to make re: peer authentication. I recall we want to stick with it, but add some notes to ensure that people are aware that's what is happening under the hood.

Should I also replicate these changes for the NodeJS path as well? I can do so in this PR as well.

Reply:
Oh this is the node path file, so I'd have to replicate to the Rails path!

Reply:
Thanks @crespire, feel free to send me a ping when you're happy for it to be reviewed 

> should I also replicate these changes for the NodeJS path as well? I can do so in this PR as well.
Yes please, these changes will be great on both paths!